src/main.js

```diff
@@ -59,9 +60,12 @@ var load = loader({
     setup: ({cfg}) => new taskcreator.TaskCreator(cfg.taskcluster),
   },
 
-  ses: {
+  notify: {
     requires: ['cfg'],
-    setup: ({cfg}) => new AWS.SES(cfg.aws.ses),
+    setup: ({cfg}) => new taskcluster.Notify({
+      credentials: cfg.taskcluster.credentials,
+      authorizedScopes: ['queue:route:notify.email.*.on-failed'],
```

Comment:
The scope should just be `...email.*`, meaning "can email anyone".

However, note from [the docs](https://docs.taskcluster.net/reference/core/taskcluster-notify/references/api#send-an-email%0Aexperimental) that the correct scope is `notify:email:*`.

BTW, I think using authorizedScopes here is a great idea, as it prevents us from doing unexpected things with this client.  "Least privilege" is a great design principle.